robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -649,23 +653,51 @@ function sendRequest (device, command, callback, timeout = REQUEST_TIMEOUT) {
   })
 }
 
+/*
+returns the IP portion of the currently set ROS_MASTER_URI
+*/
+function getRoverIP (callback) {
+  console.log('roverIP: ' + env.ROS_MASTER_IP)
+  console.log('hostIP: ' + env.HOST_IP)
+  return env.ROS_MASTER_IP
+}
+
+/*
+command sending
+*/
+function sendIKCommand (cmd) {
+  let command = new ROSLIB.Message({ data: cmd })
+  console.log(command)
+  rosLog(ROSINFO, 'Sending "' + cmd + '" to IK node')
```

Comment:
You should use `logInfo`, `rosLog` is a helper function (not meant to be called directly)

Reply:
Fixed